versioned_docs/version-3.10/use-cases/custom-pipelines-flow.md

```diff
@@ -0,0 +1,376 @@
+# Creating and Using Custom Tekton Pipelines
+
+This use case explains how to create and use custom Tekton pipelines on the KubeRocketCI Platform.
+While KubeRocketCI offers pre-configured Tekton pipelines for common use cases, custom pipelines allow you to adapt workflows to meet unique project requirements.
+It also provides guidance on integrating and using these custom Tekton pipelines within your development processes.
+
+## Goals
+
+- Provide a clear method for creating and configuring custom Tekton tasks and pipelines for build, review, deployment, and clean processes within KubeRocketCI.
+- Offer a detailed guide on integrating custom pipelines with the KubeRocketCI platform, ensuring seamless connection and functionality.
+- Clarify the process for replacing existing build, review, deployment, and clean pipelines with custom pipelines, highlighting necessary steps and considerations.
+- Demonstrate the benefits of using custom Tekton pipelines in KubeRocketCI, such as improved flexibility, scalability, and efficiency in development workflows.
+
+## Prerequisites
+
+Before proceeding with this use case, ensure the following prerequisites are met:
+
+- Access to a KubeRocketCI instance with permissions to create and edit **Components** and **Environments**.
+- A configured KubeRocketCI instance with at least one active [GitServer](https://docs.kuberocketci.io/docs/next/user-guide/add-git-server) (e.g., GitHub, GitLab or Bitbucket).
+- Configured ArgoCD instance with the [Add-ons repository](https://docs.kuberocketci.io/docs/next/operator-guide/add-ons-overview) added.
+
+## Scenario
+
+- Create a Tekton library in KubeRocketCI for storing and managing custom Tekton pipelines and trigger templates.
```

Comment:
maybe we can use 'Tekton resources' instead of providing list of all object that can be managed by this approach?